src/libs/ReportActionsUtils.ts

```diff
@@ -265,7 +265,7 @@ function getContinuousReportActionChain(sortedReportActions: ReportAction[], id?
     if (id) {
         index = sortedReportActions.findIndex((obj) => obj.reportActionID === id);
     } else {
-        index = sortedReportActions.findIndex((obj) => obj.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
```

Comment:
This function is covered by unit tests [here](https://github.com/Expensify/App/blob/ce55a7e93d6c1a90ec07e75b1796d402488506b6/tests/unit/ReportActionsUtilsTest.ts#L444-L1786). Can we please add a test to cover the edge case that we missed here? (i.e: only a created action exists in a report)

Reply:
@roryabraham Add the test.